sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -141,9 +139,9 @@ def _generate_dockerfile(filename, base_image, entrypoint_filename, python_versi
         f.write('RUN pip install -r /ml/requirements.txt\n')
     f.write('ADD ' + entrypoint_filename + ' /ml/main.py\n')
     if python_version is 'python3':
-      f.write('ENTRYPOINT ["python3 -u ", "/ml/main.py"]')
+      f.write('ENTRYPOINT ["python3 -u", "/ml/main.py"]')
```

Comment:
This won't work either. There is no program called "python3 -u".

Reply:
should be ["python3", "-u", "/ml/main.py"]

Reply:
Done